### PR TITLE
Remove references to packages.chef.io

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
## Description

[Please describe what this change achieves]

## Issues Resolved

https://progresssoftware.atlassian.net/browse/CHEF-15020

## Related PRs

- https://github.com/chef/automate/pull/8619
- chef/automate#8657
- https://github.com/chef/chef-server/pull/3920
- https://github.com/chef/chef-workstation/pull/3281
- https://github.com/chef/desktop-config/pull/534
- https://github.com/chef/desktop-config/pull/535
- https://github.com/chef/chef-workstation/pull/3287

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/packages.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The `stable` and `current` release channels support the following package reposi
 - APT (Debian and Ubuntu platforms)
 - Yum (Enterprise Linux platforms)
 
-You can download Chef Software's GPG public key from [packages.chef.io](https://packages.chef.io/chef.asc).
+You can download Chef Software's GPG public key from [chefdownload-commercial.chef.io](https://chefdownload-commercial.chef.io/chef.asc).
```
Comment (IanMadd) [inline content/packages.md]: I don't think you can do this procedure with this API. That's a question for @sonarora

File: content/install_bootstrap.md [hunk 2]
```diff
@@ -373,7 +379,7 @@ $file = "C:\Windows\System32\drivers\etc\hosts"
 $hosts | Add-Content $file
 
 ## Download the Chef Infra Client
-$clientURL = "https://packages.chef.io/files/stable/chef/12.19.36/windows/2012/chef-client-<version-here>.msi"
+$clientURL = "https://chefdownload-commercial.chef.io/files/stable/chef/12.19.36/windows/2012/chef-client-<CLIENT_VERSION>.msi"
```
Comment (IanMadd) [inline content/install_bootstrap.md]: Will this work?

**Comment by sonarora** [comment]
+ @Karunesh ***@***.***>, @Rahul ***@***.***>

@Ian ***@***.***> we need to share all PRs with all EMs to get the confirmation if all new links are correct and working before, we publish the changes in doc. I think this is not the entire list.

________________________________

File: content/packages.md [hunk 3]
```diff
@@ -41,13 +41,13 @@ To set up an APT package repository for Debian and Ubuntu platforms:
...
     ```bash
-    wget -qO - https://packages.chef.io/chef.asc | sudo apt-key add -
+    wget -qO - https://chefdownload-commercial.chef.io/chef.asc | sudo apt-key add -
     ```
 
 1. Create the APT repository source file:
 
     ```bash
-    echo "deb https://packages.chef.io/repos/apt/<CHANNEL> <DISTRIBUTION> main" > chef-<CHANNEL>.list
+    echo "deb https://chefdownload-commercial.chef.io/repos/apt/<CHANNEL> <DISTRIBUTION> main" > chef-<CHANNEL>.list
```
Comment (IanMadd) [inline content/packages.md]: It looks like this doesn't work

**Review by IanMadd — APPROVED** [review]

**Pull request merged**